salt/metalk8s/addons/dex/deployed/chart.sls

```diff
@@ -0,0 +1,139 @@
+#!jinja | kubernetes kubeconfig=/etc/kubernetes/admin.conf&context=kubernetes-admin@kubernetes
+{%- from "metalk8s/repo/macro.sls" import build_image_name with context %}
+
+{% raw %}
+
+apiVersion: v1
+kind: Secret
+metadata:
+  labels: {app.kubernetes.io/instance: dex, app.kubernetes.io/managed-by: salt, app.kubernetes.io/name: dex,
+    app.kubernetes.io/part-of: metalk8s, app.kubernetes.io/version: 2.19.0, helm.sh/chart: dex-2.4.0,
+    heritage: metalk8s}
+  name: dex
+  namespace: metalk8s-auth
+stringData: {config.yaml: "issuer: https://{{ pillar['metalk8s']['api_server']['host']\
```

Comment:
As hinted above (in the values file), this ain't right. Even adding `endraw`/`raw` there may make this fail... Not sure how to fix though.

Reply:
`endraw`/`raw` should work, don't see why it wouldn't... Jinja doesn't care about what the supposed format underneath is, IIRC.

Reply:
It could not work because of the `\`s that get injected when rendering the YAML.

Reply:
Maybe with block-style display this wouldn't need to inject `\`s... We need to test, ping @Ebaneck ;)

Reply:
I can confirm that endraw / raw does not work

Reply:
Even with block-style display, it injects `\ s..` Any other hints i could try out??